isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
Yea, I was trying to line it (which didn't really work out). But R1 had delGA and R2 has delGAGA. 

What I'm still confused about those is that it seems like the current logic would say that both agree? Since both would have empty strings covering the variant?